lib/nero/wildlife_watch/wildlife_watch_classification.rb

```diff
@@ -22,7 +22,7 @@ def annotations
 
       def task
         case workflow_id
-        when "1021"
+        when "1021", "1717"
```

Comment:
Q? are we going to move these to config files or is it better to keep these in code for the meantime?